adapters/http-vertx-base/src/test/java/org/eclipse/hono/adapter/http/AbstractVertxBasedHttpProtocolAdapterTest.java

```diff
@@ -68,6 +67,7 @@
 import org.junit.runner.RunWith;
 import org.mockito.ArgumentCaptor;
 
+import io.netty.handler.codec.http.HttpResponseStatus;
```

Comment:
I think we should use our own constant definition instead of depending on a low level library ...

Reply:
Replaced with `HttpUtils.HTTP_TOO_MANY_REQUESTS`.